tests/mock_django_classes.py

```diff
@@ -119,12 +119,18 @@ def add_segment(self, segment: DjangoSegment):
         self.segments.segments.append(segment)
 
 
+@dataclass
+class Tag:
```

Comment:
I know the idea is that we should remove these mock django classes in the future, but I think for consistency right now we should call this class `DjangoTag` ?